src/core/AutoRest.Extensions/ParameterGroupExtensionHelper.cs

```diff
@@ -111,8 +111,7 @@ private static string GenerateParameterGroupModelText(IEnumerable<Method> method
             if (methodList.Count == 1)
             {
                 Method method = methodList.Single();
-                return string.Format(CultureInfo.InvariantCulture, "Additional parameters for the {0} operation.",
-                    createOperationDisplayString(method.MethodGroup.Name.ToPascalCase(), method.Name));
+                return string.Format(CultureInfo.InvariantCulture, "Additional parameters for {0} operation.", method.Name);
```

Comment:
For consistency, shouldn't this use `SwaggerModeler.GetMethodNameFromOperationId(method.Name)` as above?
I'd just reuse `"Additional parameters for " + SwaggerModeler.GetMethodNameFromOperationId(method.Name) + " operation."` here, `string.Format` with `CultureInfo` is complete nonsense here anyways.